src/devices/bus/a2bus/a2superdrive.h

```diff
@@ -0,0 +1,22 @@
+// license:BSD-3-Clause
+// copyright-holders:Kelvin Sherlock
+/*********************************************************************
+
+    a2superdrive.h
+
+    Implementation of the Apple II 3.5 Disk Controller Card
+
+*********************************************************************/
+
+#ifndef MAME_BUS_A2BUS_SUPERDRIVE_H
+#define MAME_BUS_A2BUS_SUPERDRIVE_H
+
+#pragma once
+
+#include "a2bus.h"
+
+
+// device type definition
+DECLARE_DEVICE_TYPE(A2BUS_SUPERDRIVE, a2bus_superdrive_device)
```

Comment:
If the concrete type isn’t in the header, it shouldn’t be exposed, and it should be in an anonymous namespace to keep the global namespace cleaner.